src/main/java/frc/robot/Orchestrator.java

```diff
@@ -144,7 +144,7 @@ public Command scoreAmp() {
    */
   public Command shootBasic() {
     return Commands.sequence(
-        shooter.manualShoot(0.85).withTimeout(1),
+        shooter.manualShoot(0.85).withTimeout(5).until(() -> shooter.atVelocity(0.8)),
```

Comment:
Should this be `0.85`? Move to constant.

Or if it's supposed to be 0.8, you could do something like atVelocity(SOME_CONSTANT - 0.05)

Reply:
Fixed in orchestrator branch